src/style.css

```diff
@@ -2,2 +1,76 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.logo img {
+  width: 40px;
+  height: auto;
+  margin-left: 50px;
+  margin-top: 4px;
+}
+
+.navigation {
+  margin-right: 50px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+}
+
+.nav-item {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
```

Comment:
The `.nav-item` class uses fixed height and line-height values. This might cause issues on smaller screens or with different font sizes. Consider using more flexible units or CSS techniques to maintain alignment without fixed heights.